### PR TITLE
New package: RyzenAdj-0.2

### PR DESCRIPTION
**Pull request by maciozo**

### CODE REVIEW

File: srcpkgs/ryzenadj/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'ryzenadj'
+pkgname=ryzenadj
+version=0.2
+revision=1
+archs="x86_64 x86_64-musl"
```
Comment (maxice8) [inline srcpkgs/ryzenadj/template]: can be x86_64*

Comment (maciozo) [reply srcpkgs/ryzenadj/template]: done

File: srcpkgs/ryzenadj/template [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'ryzenadj'
+pkgname=ryzenadj
+version=0.2
+revision=1
+archs="x86_64*"
+wrksrc="RyzenAdj-${version}"
```
Comment (maxice8) [inline srcpkgs/ryzenadj/template]: the pkgname can be RyzenAdj

Comment (maciozo) [reply srcpkgs/ryzenadj/template]: I noticed that the manual said to use the same capitalisation as the dist archive file uses, but on the other hand, the binary itself is named `ryzenadj`. In my opinion, it's more intuitive if the binary and package have the same name. What do you think?

**Comment by maxice8** [comment]
On Fri, Mar 08, 2019 at 04:58:16PM -0800, maciozo wrote:
> maciozo commented on this pull request.
> 
> 
> 
> > @@ -0,0 +1,18 @@
> +# Template file for 'ryzenadj'
> +pkgname=ryzenadj
> +version=0.2
> +revision=1
> +archs="x86_64*"
> +wrksrc="RyzenAdj-${version}"
> 
> I noticed that the manual said to use the same capitalisation as the dist archive file uses, but on the other hand, the binary itself is named `ryzenadj`. In my opinion, it's more intuitive if the binary and package have the same name. What do you think?
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/9534#discussion_r263976551

I try to follow the naming of the upstream project.

**Pull request merged**